addons/website/static/src/js/tours/website_tour_reset_password.js

```diff
@@ -0,0 +1,192 @@
+odoo.define('website.tour_reset_password', function (require) {
+'use strict';
+
+var base = require('web_editor.base');
+var localStorage = require('web.local_storage');
+var rpc = require('web.rpc');
+var tour = require('web_tour.tour');
+
+var currentDomain = window.location.protocol + '//' + window.location.hostname;
+var usersUrl = '/web?debug#action=base.action_res_users&view_type=list';
+var resetLinkKey = 'website.tour_reset_password.resetLink';
+
+tour.register('website_reset_password', {
+    test: true,
+    url: '/web',
+    wait_for: base.ready(),
+},
+[
+    {
+        content: "Change the domain of the websites and go to users page",
+        trigger: 'body',
+        run: function () {
+            // We could do it with the UI but this is not the goal of this test,
+            // so we just make an RPC to be faster.
+
+            // We change the domain of the website to test that the email that
+            // will be sent uses the correct domain for its links.
+            var def1 = rpc.query({
+                'model': 'website',
+                'method': 'write',
+                'args': [[1], {
+                    'domain': "my-test-domain.com",
+                }],
+            });
+            // We need to change the domain of all the websites otherwise the
+            // website selector will return the website 2 since the domain we
+            // set on website 1 doesn't actually match our test server.
+            var def2 = rpc.query({
+                'model': 'website',
+                'method': 'write',
+                'args': [[2], {
+                    'domain': "https://domain-not-used.fr",
+                }],
+            });
+            $.when(def1, def2).then(function (data) {
+                window.location.href = usersUrl;
+            });
+        },
+    },
+    {
+        content: "click on Admin",
```

Comment:
for NEXT time,

since we dont want to test UI here, it will be better in one step to write on partner admin the correct website.
Here, if we change the webclient (td, widget class, automcomplete lib, ...)  your test will be broken, while your feature is still correct.

Reply:
ok for next time